DETAILS.md

```diff
@@ -0,0 +1,173 @@
+Salary slip kata : Additional examples and resources
+================
+
+  *You do not need to read this document to complete the Kata*
+
+  This document contains a condensed version of the calculation rules with more examples.
+
+  Note: the tables used for calculating the National Insurance contributions and Taxes are for the current year (April 2016/17).
+
+  See [Examples: National Insurance contributions and Tax calculations](#examples-national-insurance-contributions-and-tax-calculations) for further information on the breakdown.
+
+  Note: The term 'Tax-free allowance' is interchangeably used with the term 'Personal Allowance' and 'Tax-free Personal Allowance' in this domain.
```

Comment:
I would suggest to also mention this note on the README.md page, since we do use these terms on that page as well - what do you think?